contrib/intel/jenkins/tests.py

```diff
@@ -700,15 +702,15 @@ def __init__(self, jobname, buildno, testname, core_prov, fabric,
                          fabric, hosts, ofi_build_mode, user_env, log_file, mpitype,
                          util_prov)
         self.mpi_type = mpitype
-        self.mpichpath = f"{self.middlewares_path}/{self.mpi_type}_mpichtest/" \
-                         f"{self.mpi_type}_mpichsuite/"
-        self.mpichsuitepath = f'{self.mpichpath}/test/mpi/'
+        if (mpitype != 'ompi'):
+            self.mpichsuitepath = f'{self.mpi.mpichpath}/test/mpi/'
         self.pwd = os.getcwd()
         self.weekly = weekly
         self.mpichtests_exclude = {
-        'tcp'   :   {   '.'      : [('spawn','dir'), ('rma','dir')],
-                    'threads'    : [('spawn','dir'), ('rma','dir')],
-                    'errors'     : [('spawn','dir'),('rma','dir')]
+        'tcp'   :   {   '.'        : [('spawn','dir'), ('rma','dir')],
```

Comment:
There is one more test that needs to be excluded: "./threads/comm/idup_comm_gen 4"